### DISCUSSION TITLE
Migracja do Azure

### DISCUSSION THREAD

**Discussion by Martinovy** [AzureCommunityPL/group#115] [Pomoc] [answered]
Mój klient posiada rozwiązanie działające na Windows Server. Do tej pory korzystał z usług home.pl w zakresie hostingu Windows Server. Home.pl podobno zdecydowało o zakończeniu świadczenia tego rodzaju usług, więc klient musi migrować rozwiązanie. Czy ktoś jest w stanie się tego podjąć? Pomyślałem o Azure właśnie, ale nie mam doświadczenia. Sprawa dość pilna, rozliczenia bezpośrednio z klientem.

**Comment by eklime** [comment]
I nikt się nie zgłosił przez 13 godzin?
Daj znać na priv, to coś pomyślimy.

**Reply by ptrstpp950** [reply]
Jak to kiedyś @wi5nia pisał ludzie zajęci są 🤪

**Reply by wi5nia** [reply]
Bycie gwiazda interenetowych kursow jest czasochlonne @eklime wiec tak jak pisze @ptrstpp950 - brak czasu 😄

**Comment by Martinovy** [comment, marked as answer]
Późnym wieczorem dostałem rozwiązanie. Dziękuję za chęci :)
